docs/SnippetConcept.md

```diff
@@ -4,10 +4,10 @@
 
 Snippets are a helpful tool when creating templates in the FHIR Converter.
 Leveraging snippets allows you to reference them in multiple templates in the future, preventing you from having to rewrite the same code over again.
-Within the FHIR converter release, there are four types of snippets: **Resource**, **Reference**, **Data Type** and **Code System**.
+Within the FHIR converter release, there are seven types of partial templates: **Resources**, **References**, **Data Type**, **Code Systems**, **Sections**, **Utils** and **Value Set**. 
```

Comment:
`Partial templates` in handlebars are called `snippets` in dotliquid. We can still use `snippets` here.